data_structures/linked_list/doubly_linked_list.c

```diff
@@ -0,0 +1,264 @@
+/**
+ * @file    doubly_linked_list.c
+ * @brief   Doubly linked list, simple code.
+ * @details
+ * A doubly linked list is a data structure with a sequence
+ * of components called nodes. Within that nodes there are
+ * three elements: a value recorded, a pointer to the next
+ * node, and a pointer to the previous node.
+ *
+ * In this implementation, the functions of creating the list,
+ * inserting by position, deleting by position, searching 
+ * for value, printing the list, and an example of how the
+ * list works were coded.
+ *
+ * @author  [Gabriel Mota Bromonschenkel Lima](https://github.com/GabrielMotaBLima)
+ */
+#include <stdio.h>
+#include <stdlib.h>
+
+/**
+ * @brief Standard type variable for list inner values
+ */
+typedef double type;
```

Comment:
Avoid using global variables, instead, pass them
as local function parameters, or as class private members.

Reply:
Fixed up, thank you! :slightly_smiling_face: 
Great tip, by the way.